ground/gcs/src/plugins/config/configgadgetwidget.cpp

```diff
@@ -58,8 +58,11 @@ ConfigGadgetWidget::ConfigGadgetWidget(QWidget *parent) : QWidget(parent)
 {
     setSizePolicy(QSizePolicy::MinimumExpanding, QSizePolicy::MinimumExpanding);
 
+    this->setObjectName(QString("ConfigWidget"));
```

Comment:
Would it make more sense to match the name of the class to avoid yet another set of names to remember/track?